examples/autoscale/main.tf

```diff
@@ -109,49 +96,38 @@ module "vpc_trust" {
   ]
 }
 
-#-----------------------------------------------------------------------------------------------
-# Firewalls with auto-scaling.
-#-----------------------------------------------------------------------------------------------
-/*
-  Dedicated IAM service account for running GCP instances of Palo Alto Networks VM-Series.
-  The account is used for running the instances and for also for their GCP plugin access.
-*/
-
+# IAM service account for running GCP instances of Palo Alto Networks VM-Series and their GCP plugin access.
 module "iam_service_account" {
   source = "../../modules/iam_service_account/"
 
   project_id         = var.project_id
-  service_account_id = "${local.prefix}vmseries-mig-sa"
+  service_account_id = "${var.name_prefix}vmseries-mig-sa"
 }
 
 # Create VM-Series managed instance group for autoscaling
 module "autoscale" {
```

Comment:
Maybe we can add a `for_each` for the modules (not just autoscale) in the example so we can have the example flexible like the others are also done for NGFW Hub ?

Reply:
It's out of scope of this change. Let's do it as part of the actual task for NGFW Hub.